.github/workflows/main-js.yaml

```diff
@@ -53,10 +53,9 @@ jobs:
           fetch-depth: 0
 
       - name: Set up Docker Buildx
-        uses: actions/setup-node@8f152de45cc393bb48ce5d89d36b731f54556e65 # v4.0.0
-
+        uses: docker/setup-buildx-action@d70bba72b1f3fd22344832f00baa16ece964efeb # v3.3.0
```

Comment:
How it was even working without Docker 😅. I think, we can then simply remove it :)

Reply:
Yeah that's kinda odd 😅 I'm guessing the base actions runner has Docker installed but I think we'll keep it in anyway just incase that ever changes.